python/across_api/across/resolve.py

```diff
@@ -0,0 +1,128 @@
+import json
+from typing import Optional, Tuple
+
+import requests
+from astropy.coordinates.name_resolve import NameResolveError  # type: ignore
+from astropy.coordinates.sky_coordinate import SkyCoord  # type: ignore
+
+from ..base.common import ACROSSAPIBase
+from .schema import ResolveGetSchema, ResolveSchema
+
+ANTARES_URL = "https://api.antares.noirlab.edu/v1/loci"
+
+
+def antares_radec(ztf_id: str) -> Tuple[Optional[float], Optional[float]]:
+    """
+    Query ANTARES API to find RA/Dec of a given ZTF source
+
+    Parameters
+    ----------
+    ztf_id
+        ZTF name of source
+
+    Returns
+    -------
+        RA, Dec in ICRS decimal degrees, or None, None if not found
+
+    FIXME: Replace with antares-client module call in future, once confluent-kafka-python issues are resolved.
+    """
+    search_query = json.dumps(
+        {"query": {"bool": {"filter": {"term": {"properties.ztf_object_id": ztf_id}}}}}
+    )
+
+    params = {
+        "sort": "-properties.newest_alert_observation_time",
+        "elasticsearch_query[locus_listing]": search_query,
+    }
+    r = requests.get(ANTARES_URL, params=params)
+
+    if r.status_code == 200:
+        antares_data = json.loads(r.text)
+        ra = antares_data["data"][0]["attributes"]["ra"]
+        dec = antares_data["data"][0]["attributes"]["dec"]
+        return ra, dec
+    else:
+        return None, None
```

Comment:
Perhaps we should log the error as well?

Reply:
I don't really understand what you mean?

Reply:
OK, logging was something I wanted to address in a later pull request.

Reply:
Understandable, but unfortunately it's necessary here.

Reply:
c44c0ba